docs/gettingstarted.rst

```diff
@@ -0,0 +1,65 @@
+Getting started with the OMERO Python API
+ ========================================
```

Comment:
This header does not build well.
Maybe the underscore is not fully to the left as it should be ?
![Screen Shot 2020-05-01 at 17 23 03](https://user-images.githubusercontent.com/2478303/80821349-7031ed00-8bd0-11ea-95ca-b83e961586d1.png)